src/Qdrant.Client/Grpc/Expression.cs

```diff
@@ -0,0 +1,126 @@
+namespace Qdrant.Client.Grpc
+{
+    /// <summary>
+    /// To construct a formula for score boosting.
+    /// </summary>
+    public partial class Expression
+    {
+        /// <summary>
+        /// Implicitly converts a float constant into an <see cref="Expression"/>.
+        /// </summary>
+        public static implicit operator Expression(float constant) =>
+            new() { Constant = constant };
+
+        /// <summary>
+        /// Implicitly converts a variable name (string) into an <see cref="Expression"/>.
+        /// Used to reference a payload key or a score variable.
+        /// </summary>
+        public static implicit operator Expression(string variable) =>
+            new() { Variable = variable };
+
+        /// <summary>
+        /// Implicitly converts a <see cref="Condition"/> into an <see cref="Expression"/>.
+        /// Evaluates to 1.0 if true, 0.0 if false.
+        /// </summary>
+        public static implicit operator Expression(Condition condition) =>
+            new() { Condition = condition };
+
+        /// <summary>
+        /// Implicitly converts a <see cref="GeoDistance"/> into an <see cref="Expression"/>.
+        /// Represents a geographic distance in meters.
+        /// </summary>
+        public static implicit operator Expression(GeoDistance geoDistance) =>
+            new() { GeoDistance = geoDistance };
+
+        /// <summary>
+        /// Converts a date-time constant string into an <see cref="Expression"/>.
+        /// </summary>
+        public static Expression WithDateTime(string dateTime) =>
```

Comment:
Prefixed with `With` since the this class already has a `Datetime` property.

Same for others below.

Reply:
`From***` is a more idiomatic prefix (it's what Google's Well-Known types use).

```suggestion
        public static Expression FromDateTime(string dateTime) =>
```

Same for other `With***` methods in this class.